pulsar-testclient/src/main/java/org/apache/pulsar/proxy/socket/client/SimpleTestProducerSocket.java

```diff
@@ -45,6 +45,7 @@
 @WebSocket(maxTextMessageSize = 64 * 1024)
 public class SimpleTestProducerSocket {
     public static Recorder recorder = new Recorder(TimeUnit.SECONDS.toMillis(120000), 5);
+    public static Recorder cumulativeRecorder = new Recorder(TimeUnit.SECONDS.toMillis(120000), 5);
```

Comment:
I'm not sure why need to add a new field `cumulativeRecorder`.

Reply:
Consistent with other implementations such as `PerformanceProducer`, `PerformanceReader`, `ManagedLedgerWriter` and `PerformanceConsumer`

Reply:
It doesn’t seem to be used in `SimpleTestProducerSocket`

Reply:
I will remove this variable `cumulativeRecorder` and directly reuse `recorder` to get the aggregated data. @gaoran10 @congbobo184

Reply:
@congbobo184 @gaoran10 Could you take a look again?